src/confluent_kafka/schema_registry/json_schema.py

```diff
@@ -260,36 +255,37 @@ def __init__(self, schema_str, from_dict=None):
 
     def __call__(self, value, ctx):
```

Comment:
Maybe we could find a better name than `value` since it conflicts with the Kafka notion of keys and values.
I.e., can this deserializer only deserialize message values?

Reply:
let's go with 'data'